### PR TITLE
Book detail ii

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary
- Cover was added
- Flexbox was implemented

## Screenshot
book detail ii

## Trello card
https://trello.com/c/Zyqh2nzG/27-detalle-de-libro-ii-imagen

### CODE REVIEW

**Review by CrisVega08 — CHANGES_REQUESTED** [review]
Organize your branches because some of these changes we reviewed in the last PR @Juliocho990

File: scss/detail.scss [hunk 1]
```diff
@@ -1,3 +1,24 @@
+.card {
+  display: flex;
+  flex-flow: row wrap;
+  height: 100%;
```
Comment (eledwinn) [inline scss/detail.scss]: This height is really necessary? 🤔

File: scss/detail.scss [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+  max-height: 416px;
+  max-width: 900px;
+  padding: 24px;
```
Comment (eledwinn) [inline scss/detail.scss:6]: Try to round margins/paddings, example: `25px` would be nice :)

File: scss/detail.scss [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+  width: 100%;
+}
+
+.cover {
+  height: 368px;
```
Comment (eledwinn) [inline scss/detail.scss:11]: `370px` ?

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 2 above)
Comment (CrisVega08) [reply scss/detail.scss:6]: I think Julian is using 4px like base so all values are going to be a multiply of 4

File: scss/detail.scss [hunk 4]
```diff
@@ -1,3 +1,24 @@
+.card {
+  display: flex;
+  flex-flow: row wrap;
```
Comment (CrisVega08) [inline scss/detail.scss:3]: Why wrap on card?

**Pull request merged**